distribution/zip/jballerina-tools/LICENSE

```diff
@@ -178,14 +178,14 @@ broker-coordination-0.970.0.jar
 ballerina-kafka-2.0.0-java.jar                                                                      jar            apache2
 org.wso2.carbon.messaging-2.3.7.jar                                                                 bundle         apache2
 ballerina-parser-2.0.0-Preview1.jar                                                                 jar            apache2
-axiom-api-1.2.22.jar                                                                                bundle         apache2
+axiom-api-1.4.0.jar                                                                                bundle         apache2

```

Comment:
Need to add a space here align the words. There are 4 more places like this below.